src/components/DayPickerRangeController.jsx

```diff
@@ -223,6 +224,10 @@ export default class DayPickerRangeController extends React.Component {
     return isDayBlocked(day) || isOutsideRange(day) || this.doesNotMeetMinimumNights(day);
   }
 
+  isToday(day) {
+    return day.isSame(this.today, 'day');
```

Comment:
Can you change this to use the `isSameDay` helper method? It should already be included in this file I think and it protects slightly better against falsey values